src/cheatSheet_cairo.jl

```diff
@@ -25,10 +25,8 @@ function cheatsheet_cairomakie()
         stem!(axs[4], x, sin.(x); color=x)
         axs[4].title = "stem(x, y)"
 
-        linesegments!(axs[5],
-            Point2f.(vec([[x, rand()] for i = 1:2, x in rand(10)]));
-            color=1:20, colormap=:gnuplot)
-        axs[5].title = "linesegments(positions)"
```

Comment:
thanks for your suggestions. However, for linesegments, this approach was on purpose, since is not in the official docs and is the use case that I find myself the most useful. See here for example: https://beautiful.makie.org/dev/examples/generated/2d/datavis/astronauts/ and the original version actually 😄 https://github.com/lazarusA/100daysOfMakie#4-linesegments

Reply:
I think the cheatsheet is most useful for showing "what the plotting functions do".  The piecewise sinewave works great for this, because the reader can see how the same points from the previous plots are connected by segments in this plot. And using the simplest method (`linesegments(x, y)`) is also easier to understand.

It's true that this example doesn't show the most interesting way to use `linesegments`. But I don't think the cheatsheet should focus on "how to do things with this function" because anyway there are many ways to use a Makie function, and the best way depends on the use case.

What about this: we show a simple case in the cheatsheet and we add some less simple examples to the official documentation of `linesegments`. Would that be OK for you? If yes I'm happy to make a Makie PR to add some examples there.

Or if you prefer, I can replace `linesegments(x, sin.(x))` in my example with `linesegments(Point2f.(x, sin.(x))`. This uses the same method (vector of `Point2f`) as in your example.

Reply:
I've pushed another commit using `linesegments(Point2f.(x, sin.(x))`.